demomultistoreform/composer.json

```
@@ -0,0 +1,24 @@
+{
+  "name": "prestashop/demomultistoreform",
+  "description": "PrestaShop - multistore form example",
+  "homepage": "https://github.com/PrestaShop/example-modules",
+  "license": "AFL - Academic Free License (AFL 3.0)",
```

```suggestion
  "license": "AFL-3.0",
```
AFL-3.0 is [SPDX](https://spdx.org/licenses/) compatible